robert.txt

```diff
@@ -0,0 +1 @@
+i tak sie wszystko rozjebie z githubem i nie bedzie wiadomo co sie dzieje; prosze wybaczcie jak cos spierdole; it == pieklo; ja tu przyszedlem [na studia] pisac algosy, a nie uczyc sie systemu pracy w korpo; dramat; rozpacz; tragedia
```

Comment:
Te studia to jeden wielki tutorial jak być java developerem. Też nad tym ubolewam. Nie wiem, czy tak będzie jeszcze przez długi czas, ale nie powiem - staje się to trochę monotonne.
Warto chyba jednak podejść do tego jako "wiedza sama w sobie" - nie myśl, że "pracodawca będzie tego wymagał" albo że "dzięki temu się zarobi mega duży hajs, to trzeba to znać". Podejdź do tego jakbyś zdobywał nową umiejętność organizowania sobie pracy, niezwiązaną w ogóle z korpoświatem.
Oczywiście to tylko moja opinia, może trochę za bardzo romantyczna. Ten przedmiot przeżyjemy z wiedza o scrumie czy bez niej

Reply:
Dziękuję Paweł za wsparcie, spróbuję popatrzeć na to tak jak piszesz